newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionParametersEditor.js

```diff
@@ -28,10 +28,10 @@ const gd: libGDevelop = global.gd;
 type Props = {|
   project: gdProject,
   eventsFunction: gdEventsFunction,
-  eventsBasedBehavior: gdEventsBasedBehavior | null,
-  eventsBasedObject: gdEventsBasedObject | null,
-  eventsFunctionsContainer: gdEventsFunctionsContainer | null,
-  eventsFunctionsExtension: gdEventsFunctionsExtension | null,
+  eventsBasedBehavior: ?gdEventsBasedBehavior,
```

Comment:
?xxx is actually less precise because equivalent to xxx | null | undefined

It might be worth keeping xxx | null (this is what is usually done in TypeScript)

Reply:
That's what we do in Typescript but not what we do with Flow, we never use `| null` in the editor, and Flow was raising errors on `master` branch so I decided to make it consistent.

Reply:
I'm gonna check to see if I can leave it as is

Reply:
I don't know why the error disappeared. I'm gonna leave that unchanged